x/valset-pref/validator_set.go

```diff
@@ -97,6 +140,9 @@ func (k Keeper) UndelegateFromValidatorSet(ctx sdk.Context, delegatorAddr string
 		totalAmountFromWeights = totalAmountFromWeights.Add(val.Weight.Mul(tokenAmt))
 	}
 
+	// Handle rounding issue for ex: 9999.99999 = 10000
```

Comment:
Is this something to be solved in future PRs?

Reply:
nope this already does that i'll change it